compose/compose-file/compose-versioning.md

```diff
@@ -229,14 +229,15 @@ the [upgrading](#upgrading) guide for how to migrate away from these.
 
 ### Version 3.3
 
-An upgrade of [version 3](#version-3) that introduces new parameters only available
-with Docker Engine version **17.06.0+**
+An upgrade of [version 3](#version-3) that introduces new parameters only
+available with Docker Engine version **17.06.0+**
```

Comment:
I think the `+` is meant to indicate "and higher" but I'll note this explicitly.